README.md

```diff
@@ -1,4 +1,14 @@
 # Welcome to Nashville: Build an itinerary for a day trip to Music City
```

Comment:
You are a beautiful README soul and will be blessed with 1000 kiwis in your next life.

Reply:
Thank you!!